src/plays/text-avatar/components/Avatar/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { generateAbbreName, getRandomColor } from '../../hooks/utils';
+import './style.scss';
+type Props = {
+    first: string;
+    last: string;
+}
```

Comment:
Can we take this from `types.ts` file?

Reply:
I think I should keep it here since it is easier for others to read

Reply:
@hiimnhan , It will be very clean if put all types in one place.